observations/obs_converters/GSI2DART/gsi_to_dart.f90

```diff
@@ -231,15 +242,20 @@ subroutine stop2(ierror_code)
 ! adapted from GSI/src/main/stop1.f90
 
   use kinds, only: i_kind
-  use mpisetup, only : mpi_comm_world
+  use mpisetup, only : mpi_comm_world, MPI_SUCCESS
   implicit none
 
   integer(i_kind), intent(in) :: ierror_code
   integer(i_kind)             :: ierr
 
   write(*,*)'****STOP2****  ABORTING EXECUTION w/code=',ierror_code
   write(0,*)'****STOP2****  ABORTING EXECUTION w/code=',ierror_code
-  call mpi_abort(mpi_comm_world,ierror_code,ierr)
+! call mpi_abort(mpi_comm_world,ierror_code,ierr)
+  call mpi_finalize(ierr)
```

Comment:
this won't work if all tasks don't get here.  abort is the right thing to call.  shutting down is messy no matter what you try.

Reply:
Well, that's annoying ...